test/models/subnet_test.rb

```diff
@@ -144,4 +144,21 @@ class Subnet::Test < Subnet; end
   test "should have MTU set to 1500 by default" do
     assert_equal 1500, Subnet.new.mtu
   end
+
+  describe '#dns_servers' do
+    test 'should display a list of dns servers' do
+      subnet = FactoryBot.create(:subnet_ipv4, dns_primary: '8.8.8.8', dns_secondary: '8.8.4.4')
```

Comment:
I always prefer to use IPs reserved for documentation, examples and tests. https://tools.ietf.org/html/rfc5735#section-3 states:
> 192.0.2.0/24 - This block is assigned as "TEST-NET-1" for use in documentation and example code. It is often used in conjunction with domain names example.com or example.net in vendor and protocol documentation. As described in [RFC5737], addresses within this block do not legitimately appear on the public Internet and can be used without any coordination with IANA or an Internet registry. See [RFC1166].

There's also TEST-NET-2 (198.51.100.0/24) and TEST-NET-3 (203.0.113.0/24).

Reply:
As always, you are right. Let me change this.